src/BlockHorizons/PerWorldPlayer/world/database/LibasynqlWorldDatabase.php

```diff
@@ -76,7 +76,9 @@ public function save(WorldInstance $world, Player $player, PlayerWorldData $data
 			"food" => $data->food,
 			"exhaustion" => $data->exhaustion,
 			"saturation" => $data->saturation
-		]);
+		], function(int $insertId, int $affectedRows) use($world, $player) : void{
+			$player->getServer()->getLogger()->debug("Player world data successfully saved for player {$player->getName()} in {$world->getName()}.");
```

Comment:
I'd prefer if this were moved to place(s) where the plugin calls `WorldDatabase::save()` because `LibasynqlWorldDatabase` is simply an implementation of `WorldDatabase`.

Similar to `WorldDatabase::load()` method's signature (
```php
public function load(WorldInstance $world, Player $player, Closure $onLoad) : void;
```
), `WorldDatabase::save()` could accept a `Closure $onSave` that gets triggered after the data has (successfully) been written to the database.